script/test_Dockerfile

```diff
@@ -16,16 +16,24 @@ RUN apt-get update && apt-get install -y \
     python-minimal \
     --no-install-recommends
 
+# install bats 
+RUN cd /tmp \
```

Comment:
removed the trailing whitespace from this file... Per Michael's comment leaving the which/how of picking or caching a particular version of bats for another PR.